### ISSUE TITLE
Discrepancy in Sequential model example in the documentation

### ISSUE THREAD

**Issue by TheodoreGalanos** [fchollet/keras#5779]
The dimension of the input does not align with the model described in the documentation here: https://keras.io/getting-started/sequential-model-guide/

Changing the dim too 784 or the data input to 100 solves this.

Kind regards,
Theodore.

**Comment by fchollet** [comment]
Please send a PR to fix it

**Comment by nzw0301** [comment]
@TheodoreGalanos Thank you for your report. Please close this issue.

**Comment by TheodoreGalanos** [comment]
@nzw0301 Sorry I was at work. Also apologize for not sending a PR, I'm still a GitHub noobie!

Kind regards,
Theodore.
